src/mlpack/bindings/python/print_type_doc_impl.hpp

```diff
@@ -155,9 +155,13 @@ std::string PrintTypeDoc(
 {
   return "An mlpack model pointer.  This type can be pickled to or from disk, "
       "and internally holds a pointer to C++ memory containing the mlpack "
-      "model.  Note that this means that the mlpack model itself cannot be "
-      "easily inspected in Python; however, the pickled model can be loaded "
-      "in C++ and inspected there.";
+      "model.  This model pointer has 2 methods using which the parameters "
```

Comment:
```suggestion
      "model.  This model pointer has 2 methods with which the parameters "
```
Just a simple grammar fix. :)